### PR TITLE
Rename piet-wgsl to vello!

### PR DESCRIPTION
**Pull request by dfrg**
This renames `piet-wgsl` to `vello` and also merges the code from the now deleted `piet-scene` crate. There is a bit of churn, but structural and code changes have been kept to a minimum, mostly just patching up various imports and paths.

The `glyph` module carries over from `piet-scene` but I think "very soon" is a good time to drop that and expose a preliminary glyph run API in `SceneBuilder`.

We'll likely want to coordinate on how we're handling the repo (rename or fork+archive?) and then push an immediate patch to xilem to limit breakage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# piet-gpu
+# vello (formerly piet-gpu)
```
Comment (DJMcNab) [inline README.md:1]: I'd rather we had the reference to `piet-gpu` within the first paragraph, but not directly in the title.

If we do want it in the title, I think we should set an end-date on it being the title; say 2023-03-09 as that is when Rust 1.68 is released; it also happens to be exactly 3 calendar months from now.

File: vello/src/scene.rs [hunk 2]
```diff
@@ -469,3 +618,30 @@ impl<'a> PathBuilder<'a> {
         }
     }
 }
+
+fn affine_to_f32(affine: &Affine) -> [f32; 6] {
+    let c = affine.as_coeffs();
+    [
+        c[0] as f32,
```
Comment (DJMcNab) [inline vello/src/scene.rs:625]: This code can use [`[T;N]::map()`](https://doc.rust-lang.org/std/primitive.array.html#method.map). That's not important for this PR

**Review by DJMcNab — APPROVED** [review]
🚀

File: vello/src/scene.rs (hunk 2 above)
Comment (dfrg) [reply vello/src/scene.rs:625]: oh, const generics ftw!

**Pull request merged**